classes/01class/exercises/c01-aws01/fabioacp/ANSWER.md

```diff
@@ -0,0 +1,85 @@
+# C01-AWS01
```

Comment:
Also, please, rename the file to README.md instead of ANSWER.md

Reply:
done